packages/app-builder/src/components/pages/home.tsx

```diff
@@ -1,30 +1,44 @@
-import React, { FC, useEffect, useState } from 'react'
-import { useReapitConnect, ReapitConnectSession } from '@reapit/connect-session'
-import { reapitConnectBrowserSession } from '../../core/connect-session'
-import { configurationAppointmentsApiService } from '../../platform-api/configuration-api'
-import { ListItemModel } from '@reapit/foundations-ts-definitions'
-import { Title } from '@reapit/elements'
+import React, { FC, useRef } from 'react'
+import { Editor, Frame, Element } from '@craftjs/core'
+import { MainContainer } from '@reapit/elements'
+
+import { RenderNode } from '../ui/render-node'
+import Viewport from '../ui/viewport'
+import Container from '../ui/user/container'
+import Text from '../ui/user/text'
 
 export type AuthenticatedProps = {}
 
 export const Authenticated: FC<AuthenticatedProps> = () => {
-  const { connectSession } = useReapitConnect(reapitConnectBrowserSession)
-  const [appointmentConfigTypes, setAppointmentConfigTypes] = useState([] as ListItemModel[])
-
-  useEffect(() => {
-    const fetchAppoinmentConfigs = async () => {
-      const serviceResponse = await configurationAppointmentsApiService(connectSession as ReapitConnectSession)
-      if (serviceResponse) {
-        setAppointmentConfigTypes(serviceResponse)
-      }
-    }
-    if (connectSession) {
-      fetchAppoinmentConfigs()
-    }
-  }, [connectSession])
+  const iframeRef = useRef()
 
-  console.log('Appointment Config Types are: ', appointmentConfigTypes)
-  return <Title>App Builder</Title>
+  return (
+    <MainContainer>
```

Comment:
It's a small point but I think there is already a`ManContainer` in the private-route-wrapper component - you could be doubling up here! If not, consider moving it to that HOC and your pages will be wrapped by default.